aws-ssm-document/src/main/java/com/amazonaws/ssm/document/tags/TagUpdater.java

```diff
@@ -11,39 +11,94 @@
 import lombok.NonNull;
 import lombok.RequiredArgsConstructor;
 import software.amazon.awssdk.services.ssm.SsmClient;
+import software.amazon.awssdk.services.ssm.model.SsmException;
 import software.amazon.awssdk.services.ssm.model.Tag;
 import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
 
 @RequiredArgsConstructor
 public class TagUpdater {
 
+
     private static TagUpdater INSTANCE;
 
     @NonNull
     private final TagClient tagClient;
 
+    @NonNull
+    private final TagUtil tagUtil;
+
     public static TagUpdater getInstance() {
         if (INSTANCE == null) {
-            INSTANCE = new TagUpdater(new TagClient());
+            INSTANCE = new TagUpdater(new TagClient(), TagUtil.getInstance());
         }
 
         return INSTANCE;
     }
 
+    public void createTags(@NonNull final String documentName,
```

Comment:
is it used anywhere?

Reply:
This is not necessary, tag validation and creation will happen in Documents service for document creation, only updateTags is necessary for use with document updates
removed in update